static/css/components/buttonCta.less

```diff
@@ -31,7 +31,8 @@ a.cta-btn {
   }
   // stylelint-disable-next-line selector-max-specificity
   .btn-icon.read-aloud {
-    background-image: url('/static/images/icons/read aloud.svg');
+    // stylelint-disable-next-line function-url-quotes
```

Comment:
I think this disabling rule is unnecessary. We just need to use %20 to encode the space in the URL